docs/provider-configuration.md

```diff
@@ -1,29 +1,48 @@
----
-title: OpenStack cloud provider options
----
+# OpenStack Cloud Provider Configuration Options
```

Comment:
How about renaming the file to docs/README.md instead?
This file seems a main file of this doc path and it is easy to read docs on github.com if the path contains README.md.